Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -234,6 +240,16 @@ final class RewardsCollectionViewController: UICollectionViewController {
       ?|> \.isActive .~ !isHidden
   }
 
+  private func goToManagePledge(project: Project, reward: Reward, refTag _: RefTag?) {
+    let managePledgeViewController = ManagePledgeViewController.instantiate()
+    managePledgeViewController.configureWith(project: project, reward: reward)
+
+    let nav = UINavigationController(rootViewController: managePledgeViewController)
```

Comment:
Is this intentional to present using `.formSheet`? This will present managing pledge above all content like so.

<img width="959" alt="Screen Shot 2019-09-03 at 3 25 43 PM" src="https://user-images.githubusercontent.com/387596/64212707-1d484000-ce5f-11e9-9af2-d2728810cb65.png">

I kinda like it but wanted to double check that this was by design.

Also there's one implication to `iPhone XR` devices which will then display this like so

<img width="950" alt="Screen Shot 2019-09-03 at 3 25 05 PM" src="https://user-images.githubusercontent.com/387596/64212741-35b85a80-ce5f-11e9-945a-aca6d5cfab7e.png">

Maybe we only need to use `.formSheet` on iPads?

`if AppEnvironment.current.device.userInterfaceIdiom == .pad { ... }`

Reply:
Good the you pointed this out. I did a quick search and we are setting modalPresentationStyle to `.formSheet` all over the place for all devices. It would be a good project for Investment Day   to address this. :)